src/core/simulation.jl

```diff
@@ -227,20 +259,59 @@ function _build_stages!(sim::Simulation, verbose::Bool = true; kwargs...)
     return
 end
 
+function _stage_execution_count(sim::Simulation, stage_name::String; kwargs...)
+    execution_count = 0.0
+    for (key, chron) in sim.sequence.intra_stage_chronologies
+        if key.second == stage_name 
+            to_stage_res =PSY.get_forecasts_resolution(PSI.get_sys(
+                                    get_stage(sim, stage_name)))
+            from_stage_res = PSY.get_forecasts_resolution(PSI.get_sys(
+                                    get_stage(sim, key.first)))
+            to_stage_horizon = get_horizon(get_sequence(sim), stage_name)
+            to_stage_interval = get_interval(get_sequence(sim), stage_name)
+            _count = (chron.from_periods*from_stage_res - to_stage_horizon*to_stage_res + to_stage_interval) /to_stage_interval
+            if execution_count != 0.0
+                if _count != execution_count
+                    @error("Stage $stage_name has two conflicting execution counts $_count != $execution_count")
```

Comment:
Execution continues here.  Should an exception be thrown?  Same on line 289.

Reply:
Yes, I'll change this